.claude/skills/giles-voice/scripts/codesame.py

```
@@ -0,0 +1,83 @@
+    return tree
+
+
+def dump(src: str) -> str:
+    return ast.dump(strip_docstrings(ast.parse(src)), annotate_fields=True)
+
+
+if __name__ == "__main__":
+    ref = sys.argv[1]
+    bad = []
+    for path in sys.argv[2:]:
+        old = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)